src/riot-rs-threads/src/lib.rs

```diff
@@ -172,6 +179,37 @@ impl Threads {
             None
         }
     }
+
+    fn get_priority(&self, thread_id: ThreadId) -> Option<RunqueueId> {
+        self.is_valid_pid(thread_id)
+            .then(|| self.get_unchecked(thread_id).prio)
+    }
+
+    /// Changes the priority of a thread.
+    ///
+    /// Returns true if the thread is in the runqueue and the order might have
+    /// changed.
```

Comment:
```suggestion
    /// Returns the information if the scheduler should be invoked because the runqueue order 
    /// might have changed.
    /// `false` if the thread isn't in the runqueue (in which case the priority is still changed) 
    /// or if the new priority equals the current one.
```